gef.py

```diff
@@ -985,20 +985,13 @@ def __format__(self, format_spec: str) -> str:
         else:
             opcodes_len = int(format_spec[:-1])
 
-        opcodes_text = "".join("{:02x}".format(b) for b in self.opcodes[:opcodes_len])
+        opcodes_text = "".join(f"{b:02x}" for b in self.opcodes[:opcodes_len])
         if opcodes_len < len(self.opcodes):
             opcodes_text += "..."
-        return "{:#10x} {:{:d}} {:16} {:6} {:s}".format(self.address,
-                                                        opcodes_text,
-                                                        opcodes_len * 2 + 3,
-                                                        self.location,
-                                                        self.mnemonic,
-                                                        ", ".join(self.operands))
+        return f"{self.address:#10x} {opcodes_text:{opcodes_len * 2 + 3:d}s} {self.location:16} {self.mnemonic:6} {', '.join(self.operands)}"
```

Comment:
We'd like to keep lines under 100 chars. You can use the `()` thing to break this up (remember, no comma)

Reply:
oh, I thought it was 120 chars (which I also exceed in this case). I will keep it in mind.

Reply:
120 is probably fine if it's unnatural. My editor can wrap, but scrolling sideways in github itself sucks (and it's good practice in general)